base_encrypted_field/models/res_users.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    pgp_private_key = fields.Text('Private pgp key', readonly=True)
```

Comment:
@hbrunn thanks for the PoC! looks promising
I was under the impression that the private key was not leaving the client. Did I misunderstand the concept?